siatest/renter.go

```diff
@@ -59,17 +59,37 @@ func (tn *TestNode) DownloadByStream(rf *RemoteFile) (data []byte, err error) {
 
 // DownloadInfo returns the DownloadInfo struct of a file. If it returns nil,
 // the download has either finished, or was never started in the first place.
-func (tn *TestNode) DownloadInfo(lf *LocalFile, rf *RemoteFile) (*api.DownloadInfo, error) {
+// If the corresponding download info was found, DownloadInfo also performs a
+// few sanity checks on its fields.
+func (tn *TestNode) DownloadInfo(lf *LocalFile, rf *RemoteFile) (di *api.DownloadInfo, err error) {
```

Comment:
prefer not to have named variables for long functions, it's much easier for these to get lost or shadowed